src/Molecules/Duplicable/Duplicable.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import ButtonNoBorder from '../../Atoms/ButtonNoBorder'
+import FlexBox from '../../Templates/FlexBox'
+import { TrashAlt } from 'styled-icons/boxicons-solid/TrashAlt'
+
+const Duplicable = ({
+  Model,
+  addText,
+  requireContent = false,
+  instancesCount = requireContent ? 1 : 0,
+  onRemove = () => {},
+  onAdd = () => {},
+}) => {
+  const [{ lastIndex, indices }, setState] = useState({
+    lastIndex: instancesCount - 1,
+    indices:   [...Array(instancesCount).keys()],
+  })
+
+  const addDuplicata = () => {
+    setState({
+      lastIndex: lastIndex + 1,
+      indices:   [
+        ...indices,
+        lastIndex + 1,
+      ],
+    })
+    onAdd(lastIndex + 1)
+  }
+
+  const removeDuplicata = key => {
+    if (!requireContent || indices.length > 1) {
```

Comment:
this test looks duplicated with that one :

https://github.com/rgsystemes/lib-ui/blob/c972c9214f0390178507fb1b70a8b22fd5e7e2ae/src/Molecules/Duplicable/Duplicable.jsx#L45

Reply:
fixed in https://github.com/rgsystemes/rgsupv-dashboard/pull/2669/commits/c2a0cec683a8c880f80dbe7ffa1237f714dc9ee2